### PR TITLE
Hcmpre 1224 hcmpre 892

### PR DESCRIPTION
**Pull request by Shashwat12-egov**

### CODE REVIEW

File: packages/digit_data_model/lib/data/local_store/sql_store/tables/attributes.dart
Comment (naveen-egov) [inline packages/digit_data_model/lib/data/local_store/sql_store/tables/attributes.dart:1]: Need Migration script for the table alter

File: packages/digit_data_model/lib/models/entities/attributes.dart
Comment (naveen-egov) [inline packages/digit_data_model/lib/models/entities/attributes.dart:1]: Need Migration script for the table alter

**Review by naveen-egov — CHANGES_REQUESTED** [review]
Please make the Requested changes

File: packages/digit_data_model/lib/data/local_store/sql_store/tables/attributes.dart
Comment (Shashwat12-egov) [reply packages/digit_data_model/lib/data/local_store/sql_store/tables/attributes.dart:1]: Done

File: packages/digit_data_model/lib/models/entities/attributes.dart
Comment (Shashwat12-egov) [reply packages/digit_data_model/lib/models/entities/attributes.dart:1]: Done

**Pull request merged**